.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+variables:
+  RUBY_VERSION: 3.2.2
+
+# workflow:
+#   rules:
+#     - if: $CI_COMMIT_TAG
+
+build-and-release:
```

Comment:
Should we add check to this job so it triggers only when there's a new tag?
`rules:`
    `- if: $CI_COMMIT_TAG`

Reply:
Yes, the trigger should be as you say "when there is a new tag".
Right now it is manual, so we will trigger this release with a button in Gitlab... 

I thought its better to have some more control until we know it works, what do you think?

Reply:
My misunderstanding above. It will be manual and only be available when there is a new tag pushed with your proposed changes.
Thanks for clearing that up async @semirp - changes made